test/Dialect/Comb/canonicalization.mlir

```diff
@@ -1668,3 +1668,25 @@ hw.module @mul_0(in %arg0: i0, in %arg1: i0, out out: i0) {
   %0 = comb.mul %arg0, %arg1 : i0
   hw.output %0 : i0
 }
+
+// CHECK-LABEL: @assumeMuxCondInOperand
+func.func @assumeMuxCondInOperand(%arg0: i1, %arg1: i1) -> i2 {
```

Comment:
Could you add a test case that muxValue has multiple uses?

Reply:
Should be just below :smiley:

Reply:
Yeah I missed that. Thanks.